lib/data_migrate/data_migrator_five.rb

```diff
@@ -23,7 +23,7 @@ def initialize(direction, migrations, target_version = nil)
       validate(@migrations)
 
       DataMigrate::DataSchemaMigration.create_table
-      ActiveRecord::InternalMetadata.create_table
```

Comment:
Looks like this is failing because it's not available on earlier Rails versions. Maybe need to check the consts are defined to see which one to use?

This applies to the other spots too.